plugwise/helper.py

```diff
@@ -239,11 +243,98 @@ def all_locations(self):
 
         return
 
+    def all_appliances_class_finder(self, appliance, appl):
```

Comment:
Sorry, not a good name, I would propose `appliance_info_finder` or something similar as it finds: firmware version, model, name and vendor name, to be used as data for the Core `device_info`.

Reply:
reason behind the maddness: `all_appliances` function now has `all_appliances_CLASS...` and `all_appliances_TYPE...`